src/app/sprints/new-sprint/new-sprint.component.ts

```diff
@@ -17,28 +18,60 @@ export class NewSprintComponent implements OnInit {
   durationPeriodEndDate: Date;
 
   sprints: SprintDto[];
-  sprint: SprintDto;
+  newSprint: SprintDto;
+  previousSprint: SprintDto;
+  previousSprintDuration: number;
   projectId: number;
+  // sprintStateMap: Map<SprintStateDto, String>;
 
   constructor(private router: Router,
               private sprintsService: SprintsService,
               private storeService: StoreService,
               private messageService: MessageService) {
-    this.sprint = {
+    this.newSprint = {
       id: undefined,
       enrollmentPeriod: { start: '', end: '' },
       durationPeriod: { start: '', end: '' },
       dateClosed: '',
-      timeBurned: undefined,
-      timePlanned: undefined,
+      timeBurned: 0,
+      timePlanned: 0,
       sprintState: 'UPCOMING'
     };
     this.projectId = this.storeService.getCurrentProjectId();
+    // this.sprintStateMap = new Map<SprintStateDto, String>();
```

Comment:
To działa? Bo jak dla mnie to może być coś na tej zasadzie. Na backendzie jest masa mapowań rzeczy w rzeczy (`SprintState` na `SprintStateDto` i takie tam)

Reply:
"padding" -> "planowanie"?

Reply:
Już prawie działa ;)